MUICore/muicore/mui_server.py

```diff
@@ -311,7 +349,7 @@ def GetStateList(
                 else:
                     complete_states.append(s)
             else:
-                raise ValueError(f"Unknown status {state_desc.status}")
+                print(f"Unknown status {state_desc.status}")
```

Comment:
If this isn't an exception anymore, I think we should add some exclamations or something more obvious to make this visible in logs. Also maybe send to `stderr`?

Reply:
thinking about this again, I think it does make more sense to revert this back to a ValueError for now (which will also send a INTERNAL status through grpc). Then that logging PR where these messages are more clearly defined and easily read will be a priority once I get back!